velox/functions/lib/window/NthValue.cpp

```diff
@@ -19,11 +19,13 @@
 #include "velox/expression/FunctionSignature.h"
 #include "velox/vector/FlatVector.h"
 
-namespace facebook::velox::window::prestosql {
+namespace facebook::velox::functions::window {
 
 namespace {
-
+template <TypeKind OffsetTypeKind>
```

Comment:
WIndowFunctionArg class has the argType for the argument type. It should be possible to use that instead of this template variable. Please can you consider that approach.

Reply:
Sure, let me check, thanks.

Reply:
Hello @aditi-pandit ,
I changed code to use arg->type to tell if offset is integer, could you please help review again?
Thanks.